packages/apple-pay/create-apple-pay-payment-strategy.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { PaymentIntegrationService, PaymentIntegrationServiceMock } from "@bigcommerce/checkout-sdk/payment-integration";
```

Comment:
Should this file live inside `src` next to the file it is testing against?

Reply:
Good catch. It was created there by accident. Updated